style/ruby/.rubocop.yml

```diff
@@ -90,6 +90,11 @@ Metrics/AbcSize:
                  branches, and conditions.
   Enabled: false
 
+Metrics/BlockLength:
+  CountComments: false  # count full line comments?
+  Max: 25
+  ExcludedMethods: ["describe"]
```

Comment:
@tabfugnic / @thorncp / @jakecraige, decided to go with the exclusion of files in dceb15e